plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/runtime/jobs/DataSourceMonitorJob.java

```diff
@@ -315,9 +313,6 @@ public void scheduleMonitor() {
     }
 
     public static long getDisconnectTimeoutSeconds(@NotNull DBPDataSourceContainer container) {
-        if (container.getDriver().isEmbedded() && !DBWorkbench.getPlatform().getApplication().isMultiuser()) {
-            return 0;
-        }
```

Comment:
Is this removal intentional?

Reply:
Yes. It addresses https://github.com/dbeaver/dbeaver/issues/34516#issuecomment-2244771049; c24eef8575a705a87980c70413e67032a3ed2ea1